examples/all/package.json

```diff
@@ -22,18 +22,18 @@
   "devDependencies": {
     "@grammarly/focal": "file:../../",
     "@grammarly/tslint-config": "^0.3.1",
-    "@types/react": "15.0.38",
-    "@types/react-dom": "15.5.1",
+    "@types/react": "16.0.5",
+    "@types/react-dom": "15.5.4",
+    "react": "16.0.0-beta.5",
+    "react-dom": "16.0.0-beta.5",
```

Comment:
Why not RC?

Reply:
Nice catch) I will try lates third version.